### PR TITLE
Update soap to 1.0.0

### PR DESCRIPTION
**Pull request by oursodon**

### CODE REVIEW

File: lib/proxyHttpClient.js [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
+class ProxyHttpClient extends HttpClient {
+
+  constructor(sourceUri, proxyUri) {
 
-function ProxyHttpClient(sourceUri, proxyUri) {
   this.constructor.super_.apply(this);
 
   this.proxyUri = proxyUri;
   var sourceUri = sourceUri.replace(/\./g, '\\.').replace(/\//g, '\\/');
   this.sourceUriOccurrences = new RegExp('(<soap(12)?:address location=\")' + sourceUri, 'gi');
```
Comment (Errorname) [inline lib/proxyHttpClient.js:13]: 🪰 
```suggestion
    this.constructor.super_.apply(this);

    this.proxyUri = proxyUri;
    var sourceUri = sourceUri.replace(/\./g, '\\.').replace(/\//g, '\\/');
    this.sourceUriOccurrences = new RegExp('(<soap(12)?:address location=\")' + sourceUri, 'gi');
```

Comment (jguepin) [reply lib/proxyHttpClient.js:13]: ❤️

Comment (jguepin) [reply lib/proxyHttpClient.js]: Fixed in adab6d7 🙏

**Review by jguepin — APPROVED** [review]
Tested with:
- wallet creation
- money-in card
- registerIBAN
- money-out
- getWallet
- moneyInSDDInit
- transfer

**Pull request merged**